### ISSUE TITLE
Does every version can be converted

### ISSUE THREAD

**Issue by Dashi-1997** [Separius/SimCLRv2-Pytorch#1]
Hello, does fine-tuned model(such as 1% and 10%) conuld be converted to pytorch?

**Comment by Separius** [comment]
Good question! I will check it soon (probably tonight) and let you know.

**Comment by Separius** [comment]
Ok, I tested it, and it does work, but you have to change the download url and model id (250228)
I will update the code to make these works automated.

**Comment by Dashi-1997** [comment]
Thank you. There are 65 pre-training models in this article. Can all of them be converted into pytorch?Look forward to your results and updates.
![image](https://user-images.githubusercontent.com/64891953/96599305-5a50e980-1322-11eb-9a45-3288e68657d4.png)

**Comment by Separius** [comment]
Yes I have updated my internal code and it works for all of those models, I will push the changes as soon as I can.

**Comment by Dashi-1997** [comment]
Thank you very much. I hope you can keep up with the accuracy of each model on Pytorch at the same time.

**Comment by Dashi-1997** [comment]
Hello, because the article fine-tuned from a middle layer of the projection head, instead of the input layer
of the projection head as in SimCLR, do we need to change the network structure if we want to test the fine-tuned models?
